frontend/lib/features/home/account_balance.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:flutter_starter/l10n/app_localizations.dart';
+
+class AccountBalance extends HookWidget {
+  const AccountBalance({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(horizontal: 20.0),
```

Comment:
We should add some constants for these at some point so we don't have to duplicate this `20.0` everywhere in a future PR :)

Reply:
https://github.com/TBD54566975/didpay/issues/13